spring-data-cassandra/src/main/java/org/springframework/data/cassandra/repository/support/DefaultCassandraRepositoryFragmentsContributor.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.springframework.data.cassandra.repository.support;
+
+import org.springframework.data.cassandra.core.CassandraOperations;
+import org.springframework.data.cassandra.repository.query.CassandraEntityInformation;
+import org.springframework.data.repository.core.RepositoryMetadata;
+import org.springframework.data.repository.core.support.RepositoryComposition;
+
+/**
+ * Implementation of {@link CassandraRepositoryFragmentsContributor} that contributes empty fragments by default.
+ *
+ * @author Chris Bono
+ * @since 5.0
+ */
+enum DefaultCassandraRepositoryFragmentsContributor implements CassandraRepositoryFragmentsContributor {
```

Comment:
Question on naming of "default impls".... what is our team standard on this? I have seen so far quite a bit of `Foo -> FooSupport` rather than `Foo -> DefaultFoo`. I am happy to rename.

Reply:
Mostly, `SimpleFoo` or `DefaultFoo`. In JPA and MongoDB I went for `QuerydslContributor` because the name is already quite long. How about `EmptyContributor` for Cassandra?

Reply:
Yeh, I like `EmptyContributor`. I will adjust.

Reply:
Done